releasenotes/notes/0.1/dep-versions-383f4a21044cf599.yaml

```diff
@@ -0,0 +1,11 @@
+---
+features:
+  - |
+    Support for Python 3.11.
+upgrade:
+  - |
+    The minimum supported version of each Qiskit dependency has been
```

Comment:
Do we want to mention Nature here?

Reply:
Sure, I added this in 13626b82f584c5344de080c19e5ae87490899316.  Might be a little redundant from the other notes, but not a bad thing to have them all in one place, too.